### DISCUSSION TITLE
Sort by Entry Type in CP Entries Listing

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#11335] [Ideas] [answered]
It would be great to be able to click on the "entry type" column header to order a list of entries (for the selected section) by "entry type" on the entries listing pages within the control panel. It seems most columns have this functionality (for native craft entry elements) but not all. This would be a particularly useful one to enable the missing functionality for.

**Comment by brandonkelly** [comment, marked as answer]
Just added that for Craft 4.1 🎉  (d349a076d64e0419a3114e04f4b3de87b598dbf9)

**Reply by brandonkelly** [reply]
Craft 4.1 is out with this ✨
